src/Term/AliasGroup.php

```diff
@@ -35,36 +35,26 @@ class AliasGroup implements Comparable, Countable {
 	 * @throws InvalidArgumentException
 	 */
 	public function __construct( $languageCode, array $aliases = array() ) {
-		$this->setLanguageCode( $languageCode );
-		$this->setAliases( $aliases );
-	}
-
-	private function setLanguageCode( $languageCode ) {
 		if ( !is_string( $languageCode ) ) {
-			throw new InvalidArgumentException( '$languageCode must be a string; got ' . gettype( $languageCode ) );
+			throw new InvalidArgumentException( '$languageCode must be a string' );
 		}
 
 		$this->languageCode = $languageCode;
-	}
-
-	private function setAliases( array $aliases ) {
-		foreach ( $aliases as $alias ) {
-			if ( !is_string( $alias ) ) {
-				throw new InvalidArgumentException( 'Every element in $aliases must be a string; found ' . gettype( $alias ) );
-			}
-		}
-
 		$this->aliases = array_values(
-			array_filter(
-				array_unique(
-					array_map(
-						'trim',
-						$aliases
+			array_unique(
```

Comment:
Doesn't matter much. The test cases are key here. A simple way of refactoring such code is to linearize it, ...
```php
$b = fn1( $a );
$c = fn2( $b );
$d = fn3( $c );
```
... but I don't think that's worth it in this case.